webapp/src/ts/services/transitions/test-user-for-contact.transition.ts

```diff
@@ -0,0 +1,41 @@
+import { Injectable } from '@angular/core';
+
+import { Transition, Doc } from '@mm-services/transitions/transition';
+import { CreateUserForContactsService } from '@mm-services/create-user-for-contacts.service';
+
+@Injectable({
+  providedIn: 'root'
+})
+export class TestUserForContactsTransition extends Transition {
+  constructor(private createUserForContactsService: CreateUserForContactsService) {
+    super();
+  }
+
+  readonly name = 'create_user_for_contacts';
+  public readonly isEnabled = true;
+
+  init() {
+    return true;
+  }
+
+  filter(docs: Doc[]) {
+    if (!docs) {
+      return false;
+    }
+
+    return !!docs.filter(doc => doc?.create_user_for_contact === 'true').length;
+  }
+
+  async run(docs: Doc[]) {
+    if (!docs) {
+      return [];
+    }
+
+    docs = docs.filter(Boolean).map(doc => {
```

Comment:
Since multiple docs can be added/updated here in this list we need to find the ones that actually need the user created.  I think I would extract the logic from the `filter` method above and reuse it down here to filter down to just the docs we need to update.